GVRf/Framework/jni/objects/components/texture_capturer.cpp

```diff
@@ -164,8 +167,10 @@ glm::mat4 TextureCapturer::getMvpMatrix(float half_width, float half_height) {
 }
 
 void TextureCapturer::callback(int msg, char *info) {
-    Java_org_gearvrf_NativeTextureCapturer_callbackFromNative(
-            mJNIEnv, mCapturerObject, msg, info);
+    if (nullptr != mCapturerObject) {
```

Comment:
The native side makes a strong reference that it never releases (except in the special case with the listener I added). The Java object is hence never collected. If the java object is not collected, the native is not deleted either as we are dealing with a "hybrid" object.